component/org.wso2.carbon.identity.dpop/src/main/java/org/wso2/carbon/identity/dpop/token/binder/DPoPBasedTokenBinder.java

```diff
@@ -50,11 +52,11 @@
 
     private static final String BINDING_TYPE = "DPoP";
     private static final Log log = LogFactory.getLog(DPoPBasedTokenBinder.class);
-    private final List<String> supportedGrantTypes = Arrays.asList(GrantTypes.AUTHORIZATION_CODE, GrantTypes.PASSWORD,
-            GrantTypes.CLIENT_CREDENTIALS, GrantTypes.REFRESH_TOKEN);
+    static Set<String> supportedGrantTypesSet = null;
```

Comment:
shall we use `Collections.<String>emptySet();`here?

Reply:
fixed